cips/cip-pruning-window.md

```diff
@@ -29,18 +29,25 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Nodes that prune block data SHOULD store and distribute data in blocks that were created in the last 30 days.
 
+On the Celestia data availability network, both pruned and non-pruned nodes MAY advertise themselves under the existing `full` peer discovery tag, in which case the nodes MUST store and distribute data in blocks that were created in the last 30 days.
+
+Non-pruned nodes MAY advertise themselves under a new `archival` tag, in which case the nodes MUST store and distribute data in all blocks.
+
+Data availability sampling light nodes SHOULD sample blocks created in the last 30 days (the sampling window).
```

Comment:
```suggestion
Data availability sampling light nodes SHOULD sample blocks created in the last 30 days worth of seconds which is 2592000 seconds (the sampling window).
```

Reply:
Where does 2592000 come from?

This differs from 2595600 + 3600 = 2599200

Reply:
Is this resolved?

Reply:
@rootulp I've adjusted the numbers, my math was off. 

Pruned nodes should have a pruning window of 30 days + 1 hour buffer. 

Sampling window should be exactly 30 days.

Reply:
Note: the suggestion at the top of this thread still has an incorrect number.

Reply:
I've updated the suggestion above to reflect `2599200`

Reply:
> I don't understand where any of the existing second numbers come from (2599200 seconds).
> 
> 30 days = 60 * 60 * 24 * 30 = 2592000
> 30 days and 1 hour = 2592000 + 60 * 60 = 2595600

Since 30 days = 2592000 this should change to **2592000**